### PR TITLE
Strongly typed endpoint output

### PR DESCRIPTION
**Pull request by seba-aln**
refactor: Added strict typing for some customer facing elements

### CODE REVIEW

File: src/PubNub/Endpoints/Access/Audit.php [hunk 1]
```diff
@@ -88,7 +87,7 @@ protected function httpMethod()
     /**
      * @return PNAccessManagerAuditResult
```
Comment (marcin-cebo) [inline src/PubNub/Endpoints/Access/Audit.php:88]: Is this comment need when a type is defined?

File: src/PubNub/Endpoints/Objects/Channel/GetChannelMetadata.php [hunk 2]
```diff
@@ -64,11 +63,16 @@ protected function buildPath()
...
      * @return PNGetChannelMetadataResult
      */
-    protected function createResponse($result)
+    protected function createResponse($result): PNGetChannelMetadataResult
     {
         return PNGetChannelMetadataResult::fromPayload($result);
     }
 
+    public function sync(): PNGetChannelMetadataResult
+    {
```
Comment (marcin-cebo) [inline src/PubNub/Endpoints/Objects/Channel/GetChannelMetadata.php:72]: In src/PubNub/Endpoints/Objects/Channel/GetAllChannelMetadata.php there is different formatting. Sth like this:
```
public function sync(): PNGetChannelMetadataResult {
 return parent::sync();
}
```

Shouldn't it be consistent?

**Review by marcin-cebo — APPROVED** [review]

File: src/PubNub/Endpoints/Objects/Channel/GetChannelMetadata.php (hunk 2 above)
Comment (seba-aln) [reply src/PubNub/Endpoints/Objects/Channel/GetChannelMetadata.php:72]: fixed. the correct way is the opening bracket for a class or function in newline, for conditionals in same line after a single space.

Comment (marcin-cebo) [reply src/PubNub/Endpoints/Objects/Channel/GetChannelMetadata.php:72]: Did you commit this fix? I can see only one commit - initial one.

**Comment by seba-aln** [comment]
@pubnub-release-bot release

File: src/PubNub/Endpoints/Objects/Channel/GetChannelMetadata.php (hunk 2 above)
Comment (seba-aln) [reply src/PubNub/Endpoints/Objects/Channel/GetChannelMetadata.php:72]: [Yes](https://github.com/pubnub/php/pull/106/files#diff-1c017fdce7d95a53e69fc89170f87e58dd1c4296c9843fc086f305680e69e62bR80-R81)

**Review by techwritermat — APPROVED** [review]

**Pull request merged**